Sanity/pkcs11/basic/main.fmf

```diff
@@ -0,0 +1,28 @@
+summary: tests the basic pkcs#11 functionality of clevis
+component:
+  - clevis
+test: ./runtest.sh
+recommend:
+  - clevis
+  - softhsm
+  - opensc
+  - jose
+duration: 10m
+enabled: true
+tag:
+  - CI-Tier-1
+  - NoRHEL4
+  - NoRHEL5
+  - NoRHEL6
+  - TIPfail_Security
+  - Tier1
+  - Tier1security
+  - TierCandidatesFAIL
+tier: '1'
+extra-summary: /CoreOS/clevis/Sanity/pkcs11/basic
+extra-task: /CoreOS/clevis/Sanity/pkcs11/basic
+#extra-nitrate: TC#0XXXXX
+adjust:
+  - enabled: false
+    when: distro == rhel-4, rhel-5, rhel-6
```

Comment:
I would avoid also rhel-7 and rhel-8

Reply:
sure thing, fixed in the https://github.com/RedHat-SP-Security/clevis-tests/pull/8/commits/0711f411e56b6eca521842ef1fb393ceefd08eea